com.unity.render-pipelines.core/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ The version number for this package has increased due to a version update of a r
 
 ### Added
 - Support for the XboxSeries platform has been added.
+- Added SpeedTree8MaterialUpgrader for common SpeedTree8 code to support HD and Universal's ST8 shadergraphs
```

Comment:
What does "common" mean in this context? I can guess, but it could be helpful to users to be more explicit.

Reply:
How about:
"Added SpeedTree8MaterialUpgrader, which provides utilities for upgrading and importing SpeedTree 8 assets to scriptable render pipelines."

Since technically, it can be used in custom render pipelines, as long as people adhere to the same shader property names...

Reply:
Looks good to me!